tests/test_router_bgp.rb

```diff
@@ -399,12 +400,38 @@ def test_default_cluster_id
   def test_enforce_first_as
     asnum = 55
     bgp = RouterBgp.new(asnum)
-    bgp.enforce_first_as = true
-    assert(bgp.enforce_first_as,
-           'bgp enforce-first-as should be enabled')
-    bgp.enforce_first_as = false
-    refute(bgp.enforce_first_as,
-           'bgp enforce-first-as should be disabled')
+    if platform == :ios_xr
+      @device.cmd('term width 300')
```

Comment:
Is this actually needed? If so, consider making it part of basetest/ciscotest (IIRC we're already doing this with 'term length 0') so we don't end up having a bunch of piecemeal cases of this.